pyproject.toml

```diff
@@ -1,90 +1,188 @@
-[build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm[toml]>=6.2"]
-build-backend = "setuptools.build_meta"
-
 [project]
 name = "b3d"
 dynamic = ["version"]
 description = "A 3D computer vision library"
-readme = "README.md"
-requires-python = ">=3.10"
-license = {text = "Apache 2.0"}
-authors = [
-    {name = "Nishad Gothoskar", email = "nishadg@mit.edu"},
+license = { text = "Apache 2.0" }
+authors = [{ name = "Nishad Gothoskar", email = "nishadg@mit.edu" }]
+requires-python = ">= 3.10"
+dependencies = [ # py-pi pacakges (currently unavailable on conda)
```

Comment:
these are the only `pypi` dependencies now! all other deps are pulled from `conda` which is much faster. for the curious reader, `pixi` is based on `resolvo` which is a package resolver based on `MiniSat` (a binary satisfiability algo with conflict detection and learning). `resolvo` powers the `pypi` solver called `rip` alongside the `conda` solver called `rattler`. all of it is implemented in rust, and is pretty quick.